docs/samples/host-unity-webgl.md

```diff
@@ -1,36 +0,0 @@
-# Hosting a Unity WebGL game on the Internet Computer
```

Comment:
Since you're removing this, can you add a redirect rule to `/static/_redirect` to point to the new host-a-webgame page, please?

Reply:
Sure, should I simply add one line after https://github.com/dfinity/portal/blob/master/static/_redirects#L162 like below?
> /samples/host-unity-webgl /samples/host-a-webgame

Reply:
Yes, that looks right

Reply:
Fixed at https://github.com/dfinity/portal/pull/663/commits/7f83a0e6ca84e60b92eafed8854a260e6a563259